File: data-structure/Array/Check-if-an-Array-is-Sorted/javacript/Iterative.js
function isSorted(arr){
    
    let n = arr.length;
    // Iterate over the array and check if 
    // every element is greater than or
    // equal to previous element.
    for (let i = 1; i < n; i++) {
        if (arr[i - 1] > arr[i]) {
            return false;
        }
    }
    
    return true;
}

// Driver Code
let arr = [10, 20, 30, 40, 50];
let n = arr.length;

if (isSorted(arr)) {
    console.log("true");
} else {
    console.log("false");
}

File: data-structure/Array/Check-if-an-Array-is-Sorted/python/Iterative.py
def isSorted(arr):
    
    n = len(arr)
    # Iterate over the array and check if 
    # every element is greater than or
    # equal to previous element.
    for i in range(1, n):
        if (arr[i-1] > arr[i]):
            return False

    return True


if __name__ == "__main__":
  arr = [10, 20, 30, 40, 50]
  n = len(arr)
  if (isSorted(arr)):
     print("true")
  else:
     print("false")